src/poly.rs

```diff
@@ -457,3 +460,21 @@ where
         }
     }
 }
+
+/// Supplies a way to compute the barycenter of a polygon
+pub trait Barycenter
```

Comment:
In computer graphics this is generally called a [centroid](https://www.khronos.org/opengl/wiki/Type_Qualifier_(GLSL)#Interpolation_qualifiers). The "barycenter" is a bit confusing because it's similar to "barycentric".
No strong opinion on this though. How do you feel about renaming this to `Centroid`?

Reply:
No issues renaming it. Thanks!